From 1dd91e504648f94a41509cde9f04b0ec2b2ad6cf Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Fri, 12 Mar 2021 15:47:50 +0100
Subject: [PATCH] qmake.conf: lower MODULE_VERSION to 5.15.2

* to avoid incompatibility when searching for qtwebchannel:

CMake Error at /OE/build/luneos-hardknott/webos-ports/tmp-glibc/work/core2-64-webos-linux/org.webosports.app.browser/0.5.0-14+gitAUTOINC+4b5398142b-r0/recipe-sysroot/usr/lib/cmake/Qt5WebEngineCore/Qt5WebEngineCoreConfig.cmake:116 (find_package):
  Could not find a configuration file for package "Qt5WebChannel" that is
  compatible with requested version "5.15.4".

  The following configuration files were considered but not accepted:

    /OE/build/luneos-hardknott/webos-ports/tmp-glibc/work/core2-64-webos-linux/org.webosports.app.browser/0.5.0-14+gitAUTOINC+4b5398142b-r0/recipe-sysroot/usr/lib/cmake/Qt5WebChannel/Qt5WebChannelConfig.cmake, version: 5.15.2

Call Stack (most recent call first):
  /OE/build/luneos-hardknott/webos-ports/tmp-glibc/work/core2-64-webos-linux/org.webosports.app.browser/0.5.0-14+gitAUTOINC+4b5398142b-r0/recipe-sysroot/usr/lib/cmake/Qt5WebEngine/Qt5WebEngineConfig.cmake:116 (find_package)
  CMakeLists.txt:19 (find_package)

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 .qmake.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.qmake.conf b/.qmake.conf
index d5e64505..8de72145 100644
--- a/.qmake.conf
+++ b/.qmake.conf
@@ -5,4 +5,4 @@ QTWEBENGINE_OUT_ROOT = $$shadowed($$PWD)
 load(qt_build_config)
 CONFIG += warning_clean
 
-MODULE_VERSION = 5.15.4
+MODULE_VERSION = 5.15.2
